VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+
+Validators play a special role in the governance system. Being the pillars of the system, they are required to vote on every proposal. It is especially important since delegators who do not vote will inherit the vote of their validators. Each time a validator does not vote on a proposal, it will get slashed by a minimal amount.
+
+### What does staking imply?
+
+Staking Atoms can be thought of as a safety deposit on validation activities. When a validator or a delegator wants to retrieve part or all of their deposit, they send a special unbonding transaction. Then, Atoms undergo a *three weeks unbonding period* during which they are liable to being slashed for potential misbehaviors committed by the validator before the unbonding process started.
+
+Validators, and by association delegators, receive Atom provisions, fee rewards, and inherit the duty to participate in governance. If a validator misbehaves, a certain portion of its total stake is slashed (the severity of the penalty depends on the type of fault). This means that every user that bonded Atoms to this validator gets penalized in proportion of its stake. Delegators are therefore incentivized to delegate to validators that they anticipate will function safely.
+
+### Can a validator run away with its delegator’s Atoms?
+
+By delegating to a validator, a user delegates staking power. The more staking power a validator has, the more weight it has in the consensus and governance processes. This does not mean that the validator has custody of its delegator’s Atoms. *By no means can a validator run away with its delegator’s funds*. 
+
+Even though delegator’s funds cannot be stolen by their validators, they are still liable if their validators misbehave. In such case, each delegator’s stake will be partially slashed in proportion of their relative stake.
+
+### What is the probability of being chosen as a proposer for any given block? Does it go up with the quantity of Atoms staked ?
+
+The validator that is selected to propose the next block is called proposer. Each proposer is selected deterministically, and the frequency of being chosen is equal to the relative total stake (where total stake = self-bonded stake + delegators stake) of the validator. For example, if the total bonded stake across all validators is 100 Atoms and a validator's total stake is 10 Atoms, then this validator has a 10% probability to be chosen as the next proposer.
+
+### What is the incentive to stake?
+
+Each member of a validator’s staking pool earns different types of rewards:
+* **Block provisions:** Native tokens of applications run by validators (e.g. Atoms) are inflated to produce block provisions. These provisions exist to incentivize Atom holders to bond their stake, as non-bonded Atom will be diluted over time. 
+* **Transaction fees:** The Hub maintains a whitelist of token that are accepted as fee payment. 
```

Comment:
`The Cosmos Hub maintains a whitelist of tokens ...`

Reply:
ok